### ISSUE TITLE
Updating waydroid-settings fails

### ISSUE THREAD

**Issue by loveisfoss** [axel358/Waydroid-Settings#15]
Updating waydroid-settings fails when the application is already installed and the following command is run:

```
$ wget -O - https://raw.githubusercontent.com/axel358/Waydroid-Settings/main/install.sh | bash
--2023-03-26 12:19:50--  https://raw.githubusercontent.com/axel358/Waydroid-Settings/main/install.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.109.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2217 (2.2K) [text/plain]
Saving to: ‘STDOUT’

-                                             100%[==============================================================================================>]   2.17K  --.-KB/s    in 0s      

2023-03-26 12:19:51 (109 MB/s) - written to stdout [2217/2217]

invalid
/usr/share/waydroid-settings/waydroid-settings.sh exists.
[sudo] password for user: 
All set. Thanks for installing.
```

Downloading this script and then running it manually works fine. It generates the question if the user wants to upgrade waydroid-settings and then the user gets to chose yes or no.

```
$ wget https://raw.githubusercontent.com/axel358/Waydroid-Settings/main/install.sh
$ sh /location/to/install.sh
```

**Comment by axel358** [comment]
Can u still reproduce this on the latest version?

**Comment by loveisfoss** [comment]
No, it seems like this is fixed now, probably by the commit 196944f922e58c9e07e309049a99a4b67fcf92a9. I mark this as closed, I hope it is okay.
